js/index.js

```diff
@@ -711,11 +420,13 @@ $(function() {
                     setupEditor();
                 }).fail(function() {
                     console.log("Unable to load gist contents");
+                    $('#code_textarea').val("-- Unable to load gist contents. Gist = " + id);
```

Comment:
I doesn't have to - just meant that it currently does, because of https://github.com/purescript/trypurescript/blob/gh-pages/js/index.js#L375

I can try and make this behaviour more explicit if you prefer (e.g. adding a `isRetry` flag or something to the functions)